microsoft/testsuites/display/drm_sanity.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import Any, cast
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+
+from lisa.sut_orchestrator import AZURE, READY
+from lisa.base_tools import Cat
+from lisa.tools import Lsmod
+
+@TestSuiteMetadata(
+    area="drm",
+    category="functional",
+    description="""
+    This test suite uses to verify drm driver sanity.
+    """,
+    requirement=simple_requirement(supported_platform_type=[AZURE, READY]),
+)
+class drmSanity(TestSuite):
```

Comment:
The name can be like `Drm`, it's right for Python. And if there is no more test to added, it doesn't need to add "Sanity" with the name.